docs/Building.md

```diff
@@ -118,42 +115,42 @@ and llvm.
+The `build-mlir-aie-pcie.sh` script will automatically build a local install of the ROCm Runtime and aie-rt if the corresponding install directories are not provided. Another option is to build these externally and then point `build-mlir-aie-pcie.sh` to where they are installed. Below are the steps for how to build the tools using this second option.
+
+We chose to install the aie-rt library in /opt/xaiengine but it is not required for the tools to be installed there. Just ensure that when building mlir-aie and mlir-air, that you point to the directory in which the aie-rt library was installed. Below are the steps to build the aie-rt library and move the installation to /opt/xaiengine.
 
 ```
 git clone https://github.com/stephenneuendorffer/aie-rt
 cd aie-rt
 git checkout phoenix_v2023.2
 cd driver/src
 make -f Makefile.Linux CFLAGS="-D__AIEAMDAIR__"
-sudo cp -r ../include /opt/aiengine/
+sudo cp -r ../include /opt/xaiengine/
 sudo cp libxaiengine.so* /opt/xaiengine/lib/
 export LD_LIBRARY_PATH=/opt/xaiengine/lib:${LD_LIBRARY_PATH}
 ```
 
-When targetting the VCK5000 Versal device, you must build and install our experimental ROCm runtime (ROCr) which allows us to communicate with the AIEs. The [ROCm-air-platforms](https://github.com/Xilinx/ROCm-air-platforms) repository contains documentation on how to install ROCr. Run the following script to clone the ROCm-air-platforms repository:
+To install our experimental ROCm runtime  which allows us to communicate with the AIEs on the VCK5000, it is necessary to [install a global version of ROCM 5.6](https://rocm.docs.amd.com/en/docs-5.6.0/deploy/linux/os-native/install.html). Details of all these steps can be found in the [ROCm-air-platforms](https://github.com/Xilinx/ROCm-air-platforms#getting-started). Afterwards, follow the steps in [ROCm-air-platforms](https://github.com/Xilinx/ROCm-air-platforms#getting-started) to build the experimental ROCm runtime that targets the AIEs in the VCK5000. You can run the following script to clone the ROCm-air-platforms repository:
 
 ```
 ./utils/clone-rocm-air-platforms.sh
 ```
 
-Then, set `${ROCM_ROOT}` to the ROCm install from the previous path. Then, run the following command to build the mlir-aie toolchain targetting the VCK5000.
+Then, set `${ROCM_ROOT}` to the ROCm install. Then, run the following command to build the mlir-aie toolchain targetting the VCK5000 pointing to the externally installed aie-rt and experimental ROCm runtime.
 
 ```
-./utils/build-mlir-aie-pcie.sh llvm/build/ build install /opt/xaienginev2 ${ROCM_ROOT}/lib/cmake/hsa-runtime64/ ${ROCM_ROOT}/lib/cmake/hsakmt/
+./utils/build-mlirgaie-pcie.sh llvm/build/ build install /opt/xaiengine ${ROCM_ROOT}/lib/cmake/hsa-runtime64/ ${ROCM_ROOT}/lib/cmake/hsakmt/
```

Comment:
I guess there is a typo here in `./utils/build-mlirgaie-pcie.sh` => `./utils/build-mlir-aie-pcie.sh`.

Reply:
Thanks for catching! Addressed in [4ce47c8](https://github.com/Xilinx/mlir-aie/pull/1597/commits/4ce47c857d36eb4a344600c0b16b175ee7792ffc).